interface/src/lib/helpers/constants.ts

```diff
@@ -59,7 +59,52 @@ export const DownColorText = "text-red-600"
 export const FocusUpColorBg = "focus:bg-green-500"
 export const FocusDownColorBg = "focus:bg-red-500"
 
-export const CoinsTypeTabs = ["favorites", "all", "mainstream", "layer1", "layer2", "inscription", "ai", "meme", "defi", "game_fi", "nft"]
+export const CoinsTypeTabs = [
```

Comment:
@zed-wong Do we want to download up to date list from coingecko api instead (it woluld be too many choices than to fit them all on the screen).

Favourites and mainstream does not seem to be existing categories. At least not the one I was able to spot on the list.

Reply:
I think the current whitelisted data list is alright, we won't need the up to date list. 

`Favourites` is a list that shows all tokens a user have `star` before. Which is not implemented in backend, i think we can skip this for now. 
`Mainstream` is a placeholder, don't mind it :)